fdmi/plugins/motr-fdmi-app.md

```diff
@@ -20,106 +20,100 @@ The FDMI source instances send matched records to the FDMI Application or Plugin
 
 For each record, the FDMI Application or Plugin performs actions and sends acknowledgements back to the source instances using Motr RPC calls.
 
-## Create and Test a "Hello" FDMI sample Application 
+## Create and Test a "Hello" FDMI sample Application
 
 1.  Follow the [motr QSG guide](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) in cortx-motr repo to build Motr.
 
 2.  Follow the [hare QSG guide](https://github.com/Seagate/cortx-hare/blob/main/README.md) in the cortx-hare repo to get a Motr cluster up and running.
 
     2.1. Need to edit the config file (CDF) to add a filter. Examples of CDF file can be found [here](https://github.com/Seagate/cortx-hare/blob/25274c3787b1adab1eac3a4ee5b9fef80035ebc1/cfgen/examples/singlenode.yaml#L51). In our "Hello" example, we need to modify the fdmi_filters section to be as follow:
```

Comment:
This should be replaced with:
s/https://github.com/Seagate/cortx-hare/blob/25274c3787b1adab1eac3a4ee5b9fef80035ebc1/cfgen/examples/singlenode.yaml#L51/
https://github.com/Seagate/cortx-hare/blob/main/cfgen/examples/singlenode.yaml#L51

Reply:
Why? CDF file may change with the time and the link will point to an incorrect place.

Try your suggested link - it already points to the wrong line.

Reply:
I also suggest to add main here.